src/components/Navbar.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import logo from '../assets/planet.png';
+import './styles/Navbar.css';
+
+function Navbar() {
+  return (
+    <nav>
+      <div className="logo-container">
+        <img className="logo" src={logo} alt="logo" />
+        <h1>Space Travelers&apos; Hub</h1>
+      </div>
+      <ul className="navItems">
+        <li>
+          <NavLink to="/" className="nav-link">Rockets</NavLink>
+        </li>
+        <li>
+          <NavLink to="/missions" className="icon nav-link">Missions</NavLink>
+        </li>
+        <li>
+          <NavLink to="/profile" className="nav-link">My Profile</NavLink>
+        </li>
+      </ul>
+    </nav>
```

Comment:
- [ ] @Team great work on your navbar however, there is a slight difference between your link's color and the color given in the design, the same applies to the reserve rockets button, check the comparison below for better understanding:

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/meryemsanem/react-capstone/assets/61408860/7c5ece72-40bc-490c-8cc0-5e3fe49dd275)  |  ![image](https://github.com/meryemsanem/react-capstone/assets/61408860/c2df615a-6600-4d48-9a22-a7489d0301c3)